src/main/java/edu/ie3/datamodel/io/source/IdCoordinateSource.java

```diff
@@ -75,6 +74,94 @@
         GeoUtils.calcOrderedCoordinateDistances(
             coordinate,
             (coordinates != null && !coordinates.isEmpty()) ? coordinates : getAllCoordinates());
-    return sortedDistances.stream().limit(n).toList();
+    return restrictToBoundingBoxWithSetNumberOfCorner(coordinate, sortedDistances, n);
+  }
+
+  /**
+   * Method to turn a distance into a latitude and longitude deltas. The methode can be found here:
+   * <a href="https://math.stackexchange.com/questions/474602/reverse-use-of-haversine-formula">
```

Comment:
Link is missing a closing tag

Reply:
See https://github.com/ie3-institute/PowerSystemUtils/pull/321